<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Print</class>
 <widget class="QGroupBox" name="Print">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>知识编号</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>知识内容</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>结果编号</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>结果内容</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>知识编号</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>推理过程</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>推理结果</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Knw_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加知识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Knw_delete">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除知识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Result_add">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Result_delete">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rule_add">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加规则</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rule_delete">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除规则</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Renew">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
